### ISSUE TITLE
“It looks like your Babel configuration specifies a module transformer” when using transform-decorators-legacy

### ISSUE THREAD

**Issue by flying-sheep** [rollup/rollup-plugin-babel#18]
but i only see in its package.json

```json
"dependencies": {
  "babel-plugin-syntax-decorators": "^6.1.18",
  "babel-runtime": "^6.2.0",
  "babel-template": "^6.3.0"
},
```

no idea why that error appears

**Comment by Rich-Harris** [comment]
Bizarre. Turns out `transform-decorators-legacy`, for reasons best known to itself, translates `export default Foo` to `export { Foo as default }`. The pre-flight check that ensures modules aren't transformed is now aware of that possibility. #20 fixes it.

**Comment by Rich-Harris** [comment]
Released as 2.3.0

**Comment by flying-sheep** [comment]
:+1:
